rio_tiler/io/stac.py

```diff
@@ -118,15 +120,15 @@ class STACReader(MultiBaseReader):
 
     Args:
         filepath (str): STAC Item path, URL or S3 URL.
-        item (Dict or pystac.Item, STAC): Stac Item.
-        minzoom (Optional[int]): Set minzoom for the tiles. Defaults to tms.minzoom.
-        minzoom (Optional[int]): Set maxzoom for the tiles. Defaults to tms.maxzoom.
-        include (Optional[Set[str]]): Only Include specific assets. Defaults to None.
-        exclude (Optional[Set[str]]): Exclude specific assets. Defaults to None.
-        include_asset_types (Optional[Set[str]]): Only include some assets base on their type. Defaults to None.
-        exclude_asset_types (Optional[Set[str]]): Exclude some assets base on their type. Defaults to None.
-        reader (BaseReader): rio-tiler Reader (default is set to rio_tiler.io.COGReader).
-        reader_options (dict): additional option to forward to the Reader (default is {}).
+        item (dict or pystac.Item, STAC): Stac Item.
+        minzoom (int, optional): Set minzoom for the tiles. Defaults to tms.minzoom.
```

Comment:
I kinda like the `Optional[int]` syntax, but ok

Reply:
seems that using `type` notation is not yet standard. we can update this later